docs/sources/release-notes/v2.2.md

```diff
@@ -0,0 +1,43 @@
+
+Grafana Labs is excited to announce version 2.2 of Grafana Mimir, the most scalable, most performant open source time series database in the world.
+
+Below we highlight the top features, enhancements and bugfixes in this release, as well as relevant callouts for those upgrading from Grafana Mimir 2.1.
+The complete list of changes is recorded in the [Changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Support for ingesting out-of-order samples**: Grafana Mimir introduces new experimental support for ingesting out-of-order samples. Feature is enabled by configuring the time window since the most-recently ingested sample for the series. This can be enabled on per-tenant basis.
+
+- **Error messages**: We have overhauled the error messages reported to end user to make them easier to understand, and to include error codes that can be easily searched for. This effort will continue in next releases as well.
+
+- **Prefixed bucket store**: It is now possible to store blocks, rules and alerts into a single bucket, by configuring bucket prefix for each feature.
+
+- **Backfill API**: Grafana Mimir 2.2 introduces new API for backfill. Work on client is in progress. (Shall we drop this until client is available?)
+
+## Upgrade considerations
+
+We have updated default values and some parameters in Grafana Mimir 2.2 to give users better out-of-the-box experience:
+
+- Message size limits for gRPC messages exchanged between internal Mimir components has been increased to 100 MiB.
+
+- `-blocks-storage.bucket-store.ignore-blocks-within` parameter has been changed from `0` to `10h`. Default value of `-querier.query-store-after` has been changed from 0 to `12h`. Both changes improve query-performance for most-recent data by querying only ingesters.
+
+- Ingester's `-ingester.ring.join-after` is no longer respected, and ingester treats it as 0.
+
+- If you use shuffle sharding and configure lookback period in queriers and ruler, note that option `-querier.shuffle-sharding-ingesters-lookback-period` has been deprecated, and replaced with `-querier.shuffle-sharding-ingesters-enabled`. Lookback period now equals to value specified by `-querier.query-ingesters-within` option.
+
+- When using memberlist and component fails to join the memberlist cluster, it no longer aborts startup by default: `-memberlist.abort-if-join-fails` parameter now defaults to false.
+
+## Bug fixes
+
+- [PR 1883](https://github.com/grafana/mimir/pull/1883): Fixed a bug that caused Grafana Mimir to crash when it received user-query with special regular expression label matcher.
+- [PR 1933](https://github.com/grafana/mimir/pull/1933): Fixed a bug in ring page exposed by Ingesters, which showed incorrect status of entries in the ring.
+- [PR 2090](https://github.com/grafana/mimir/pull/2090): When ruler uses remote rule evaluation, it now applies timeout correctly. Previously it could get stuck forever and stop rule evaluation.
```

Comment:
```suggestion
- [PR 2090](https://github.com/grafana/mimir/pull/2090): Ruler in [remote rule evaluation mode](link) now applies timeout correctly. Previously it could get stuck forever, effectively halting rule evaluation.
```

Link to https://grafana.com/docs/mimir/next/operators-guide/architecture/components/ruler/#remote